app/content/models/registration.py

```diff
@@ -237,3 +237,8 @@ def get_waiting_number(self):
                 if self == waiting:
                     return list(self.event.get_waiting_list()).index(self) + 1
         return None
+
+    @property
+    def submissions(self):
```

Comment:
This property is called `submissions`, but returns forms.

Reply:
Very nicely spotted!